### DISCUSSION TITLE
Planning a contributor's specific TG channel

### DISCUSSION THREAD

**Discussion by SunWeb3Sec** [SunWeb3Sec/DeFiHackLabs#45] [Ideas]
**Goal:** Tech discuss, learn from each one.  Group a web3 security labs. we can participate CTF, bug bounty or create a whitehat DAO, etc.

**Who can join:** Contributors  in this repo.

Any ideas are welcome.

@eugenioclrc @finn79426 @0xDatapunk @zhouxianyuan @eugenioclrc @zzzuhaibmohd @m0ham3dx @softlinkprotocol @mattmitchell97 @Cache-and-Burn @xlk3099 @bentobox19 @lmanini @Caijiji

**Comment by lmanini** [comment]
Would be a great resource for all! I'd add that there we could coordinate to form a team and participate in audit contests.
Depending on number of members we should consider making a tg group or a discord server 🏎️

**Comment by Cache-and-Burn** [comment]
Awesome, sounds like a great idea. Would love to be involved.

**Reply by m0ham3dx** [reply]
-moved to correct thread

**Comment by m0ham3dx** [comment]
Thanks for the tag , making a collective is a great idea, especially for skill, knowledge and tactical exchange, I fully support it and am in favor of a Discord group, 😁.  Reasoning for discord over telegram is because , dont want my phone to be inundated with many unread messages, but that is just a personal opinion.

**Comment by eugenioclrc** [comment]
Same feelings here, it would be nice to have some place to join forces to
discuss latest hacks and collaborate to build more POC. I think discord
woulbe nicer, because its easy to share code

El jue, 29 sept 2022 a las 11:05, m0ham3dx ***@***.***>)
escribió:

> Thanks for the tag , making a collective is a great idea, especially for
> skill, knowledge and tactical exchange, I fully support it and am in favor
> of a Discord group, 😁. Reasoning for discord over telegram is because ,
> dont want my phone to be inundated with many unread messages, but that is
> just a personal opinion.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SunWeb3Sec/DeFiHackLabs/discussions/45#discussioncomment-3763188>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACRFCQBCUGRTJJAC336K7LWAWO3XANCNFSM6AAAAAAQYWUMZY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*May the source be with you.*

**Comment by SunWeb3Sec** [comment]
Cool, discord is okay from me. I will find a time to setup a discord server.

**Comment by SunWeb3Sec** [comment]
Guys, feel free to join our discord server.

please DM me to ask discord link

I will manually grant the specific roles to you.

**Reply by SunWeb3Sec** [reply]
@0xDatapunk  @m0ham3dx @softlinkprotocol @mattmitchell97 @Cache-and-Burn @xlk3099 @bentobox19 @lmanini @CaiJiJi

Anyone has not joined the discord?

**Reply by Delvir0** [reply]
Would love to see a channel where hacks are being explained to eachother!

**Reply by LucasMartinCalderon** [reply]
Count me in!

**Comment by LucasMartinCalderon** [comment]
Count me in!

**Reply by SunWeb3Sec** [reply]
https://discord.gg/wNTc5QYN

**Comment by advock** [comment]
heyyy is there any another discord group or channel specially for contributers?

**Reply by SunWeb3Sec** [reply]
Yes, all contributors are granted whitehat role in discord.

**Reply by SunWeb3Sec** [reply]
Yes, we have Whitehat private channel.
